src/storage/src/hummock/mod.rs

```diff
@@ -232,16 +231,34 @@ impl HummockStorage {
                 shutdown_sender: event_tx.clone(),
             }),
             storage_core,
-            version_update_notifier_tx: hummock_event_handler.version_update_notifier_tx(),
-            seal_epoch: hummock_event_handler.sealed_epoch(),
+            version_update_notifier_tx,
+            seal_epoch,
             hummock_event_sender: event_tx,
+            read_version_mapping,
         };
 
-        tokio::spawn(hummock_event_handler.start_hummock_event_handler_worker());
-
         Ok(instance)
     }
 
+    // TODO: make it self function and remove hummock_event_sender parameter
+    async fn new_local(
+        table_id: TableId,
+        id: u64,
```

Comment:
Are we planning to use this provided `id` for deduplication? If not, can we just create a new local instance with a self-generated id on every `new_local` call?